src/modm/platform/timer/stm32/general_purpose.cpp.in

```diff
@@ -197,6 +197,89 @@ modm::platform::Timer{{ id }}::configureOutputChannel(uint32_t channel,
 	}
 }
 
+void
+modm::platform::Timer{{ id }}::configureOutputChannel(uint32_t channel,
+OutputCompareMode mode, Value compareValue, 
+PinState out, OutputComparePolarity polarity,
+PinState out_n, OutputComparePolarity polarity_n,
+OutputComparePreload preload)
+{
+	channel -= 1;	// 1..4 -> 0..3
+
+	// disable output
+	TIM{{ id }}->CCER &= ~(0xf << (channel * 4));
+
+	setCompareValue(channel + 1, compareValue);
```

Comment:
can we just call the `configureOutputChannel` function from below here after the compare value was set? (DRY)